src/fontra/views/fontinfo/panel-sources.js

```diff
@@ -428,13 +455,62 @@ class SourceBox extends HTMLElement {
     }
   }
 
-  toggleShowHide() {
-    const element = this.querySelector("#open-close-icon");
-    element.classList.toggle("item-closed");
+  toggleShowHide(altKey) {
+    const cardElements = !altKey
+      ? [this]
+      : document.querySelectorAll(".fontra-ui-font-info-sources-panel-source-box");
+
+    const thisIconElement = this.querySelector("#open-close-icon");
+    const isClosed = thisIconElement.classList.contains("item-closed");
+
+    for (const cardElement of cardElements) {
+      const elementIcon = cardElement.querySelector("#open-close-icon");
+      if (isClosed) {
+        cardsInfos[cardElement.sourceIdentifier]["isClosed"] = false;
```

Comment:
```suggestion
        cardsInfos[cardElement.sourceIdentifier].isClosed = false;
```

Same below, please look for more occurances.